src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -67,7 +67,7 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
         <AssetActions
           assetId={assetId}
           accountId={accountId ? accountId : singleAccount}
-          cryptoBalance={cryptoBalance}
+          cryptoBalance={cryptoBalance ?? '0'}
```

Comment:
might be nice to standardize fallback value on selectors next to the `useAppSelector` call so you can know top level what you are dealing with